apps/graph/list/color_list.h

```diff
@@ -0,0 +1,12 @@
+#pragma once
```

Comment:
```suggestion
#ifndef GRAPH_COLOR_LIST_H
#define GRAPH_COLOR_LIST_H
```
This should be done in all header